packages/graph/hash_graph/lib/graph/src/store/postgres/query/condition.rs

```diff
@@ -261,8 +261,8 @@ mod tests {
                     Cow::Borrowed("right"),
                 ))),
             )]),
-            r#"("data_types"."schema"->>'left' = "data_types"."schema"->>'right')"#,
-            &[],
+            r#"("data_types"."schema"->>$1 = "data_types"."schema"->>$2)"#,
```

Comment:
Is this still without parameters then? And we aren't calling this `render` anymore, correct?